locations/storefinders/uberall.py

```diff
@@ -54,9 +56,9 @@ def parse(self, response: Response):
 
             yield from self.post_process_item(item, response, feature)
 
-    def post_process_item(self, item: Feature, response, location: dict):
+    def post_process_item(self, item: Feature, response: Response, location: dict) -> Iterable[Feature]:
         """Override with any post-processing on the item."""
         yield item
 
-    def pre_process_data(self, location, **kwargs):
```

Comment:
Do we need the implicit None?

Reply:
Probably not necessary, but I don't think it hurts.